config/routes.rb

```diff
@@ -5,11 +5,16 @@
   defaults format: :json do
     resources :project, except: [:new, :edit]
     resources :task, except: [:new, :edit, :destroy]
+    namespace :single_project do
+      namespace :id do
+       resources :users, only: [:index]
```

Comment:
set this under `resources :project` and use `member: true` to generate more elegant routing